hack/.ci/run-e2e-openshift-aws.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2023 ScyllaDB
+#
+
+set -euExo pipefail
+shopt -s inherit_errexit
+
+trap 'kill $( jobs -p ); exit 0' EXIT
+
+if [ -z "${ARTIFACTS+x}" ]; then
+  echo "ARTIFACTS can't be empty" > /dev/stderr
+  exit 2
+fi
+
+source "$( dirname "${BASH_SOURCE[0]}" )/../lib/kube.sh"
+source "$( dirname "${BASH_SOURCE[0]}" )/lib/e2e.sh"
+parent_dir="$( dirname "${BASH_SOURCE[0]}" )"
+
+trap gather-artifacts-on-exit EXIT
+
+REENTRANT="${REENTRANT=false}"
+export REENTRANT
+
+SO_INSTALL_PROMETHEUS_OPERATOR="true"
+export SO_INSTALL_PROMETHEUS_OPERATOR
+
+SO_NODECONFIG_PATH="${SO_NODECONFIG_PATH=${parent_dir}/manifests/cluster/nodeconfig-openshift-aws.yaml}"
+export SO_NODECONFIG_PATH
+SO_CSI_DRIVER_PATH="${SO_CSI_DRIVER_PATH=${parent_dir}/manifests/namespaces/local-csi-driver/}"
+export SO_CSI_DRIVER_PATH
+SO_SCYLLACLUSTER_STORAGECLASS_NAME="${SO_SCYLLACLUSTER_STORAGECLASS_NAME=scylladb-local-xfs}"
+export SO_SCYLLACLUSTER_STORAGECLASS_NAME
+
+SCYLLA_OPERATOR_FEATURE_GATES="${SCYLLA_OPERATOR_FEATURE_GATES:-AllAlpha=true,AllBeta=true}"
+export SCYLLA_OPERATOR_FEATURE_GATES
+
+for i in "${!KUBECONFIGS[@]}"; do
+  KUBECONFIG="${KUBECONFIGS[$i]}" DEPLOY_DIR="${ARTIFACTS}/deploy/${i}" timeout --foreground -v 10m "${parent_dir}/../ci-deploy.sh" "${SO_IMAGE}" &
```

Comment:
we should either add a dedicated script running `ci-deploy-release.sh` and set it up in CI or at the very least have a tracking issue

Reply:
I vote for the issue, but I can't technically create it, would you mind?

Reply:
for openshift? the manifests are shared

Reply:
Is the split for GKE only coming from the fact `ci-deploy.sh` was there before the periodics or is there a different reason that's not there for openshift?

Reply:
`ci-deploy-release.sh` uses the manifest from a real commit (which is not there for PRs because it's a local merge, so they use `ci-deploy.sh`, by design.

`run-e2e-openshift-aws.sh` is the entrypoint for presubmits (actually wired for periodics as well at this point)
`run-e2e-openshift-aws-release.sh` needs to be added later on and wired for postsubmits and periodics

Reply:
Hence my comment - let's at least have a tracking issue in that case

Reply:
Given the situation, it's hard for me to open an issue that will get closed tomorrow :(